README.md

```diff
@@ -71,9 +71,9 @@ package main
 import (
 	"fmt"
 
-	"github.com/TAAL-GmbH/arc/api"
-	apiHandler "github.com/TAAL-GmbH/arc/api/handler"
-	"github.com/TAAL-GmbH/arc/api/transactionHandler"
+	"github.com/bitcoin-sv/arc/api"
```

Comment:
If this example code is changed, then also all the references in the code have to be changed. That starts with changing the first line in `go.mod` from `module github.com/TAAL-GmbH/arc` to `module github.com/bitcoin-sv/arc`.
Thus I'd either change all the paths in the code too or revert the change in this example